### PR TITLE
Fixed a regression when standalone widgets wouldn't work

### PR DESCRIPTION
**Pull request by vxsx**
The regression happened because in
https://github.com/divio/django-cms/commit/a44cf25d6d5fd63cd2e9bd2ad1b5e14ae15c7d7b
and before that in 3.3.0 the files required for widgets were
removed from media js class on the assumption that the CMS provided
bundles would be loaded anyway. This commit changes the way
requirements are loaded for those widgets so they can be used
standalone and on CMS pages as well.

https://github.com/divio/django-cms/issues/5476

### CODE REVIEW

**Comment by lory87** [comment]
lgtm

File: cms/forms/widgets.py [hunk 1]
```diff
@@ -24,8 +24,9 @@ class PageSelectWidget(MultiWidget):
 
     class Media:
         js = (
```
Comment (czpython) [inline cms/forms/widgets.py]: please change to:

```
js = (
    static_with_version('cms/js/widgets/forms.pageselectwidget.js'),
    static_with_version('cms/js/dist/bundle.forms.pageselectwidget.min.js'),
)
```

File: cms/forms/widgets.py [hunk 2]
```diff
@@ -123,9 +126,9 @@ class Media:
             )
         }
         js = (
-            'cms/js/select2/select2.js',
-            'cms/js/widgets/forms.pagesmartlinkwidget.js',
+            'cms/js/dist/bundle.forms.pagesmartlinkwidget.min.js',
         )
+        js = tuple(map(static_with_version, js))
```
Comment (czpython) [inline cms/forms/widgets.py]: same as above

File: cms/forms/widgets.py [hunk 3]
```diff
@@ -191,8 +196,9 @@ class AppHookSelect(Select):
 
     class Media:
         js = (
-            'cms/js/widgets/forms.apphookselect.js',
+            'cms/js/dist/bundle.forms.apphookselect.min.js',
         )
+        js = tuple(map(static_with_version, js))
```
Comment (czpython) [inline cms/forms/widgets.py]: same as above

File: cms/forms/widgets.py [hunk 4]
```diff
@@ -244,8 +250,9 @@ class ApplicationConfigSelect(Select):
...
```
Comment (czpython) [inline cms/forms/widgets.py]: same as above

File: cms/forms/widgets.py (hunk 1 above)
Comment (vxsx) [inline cms/forms/widgets.py]: sure, don't really need both of them though

**Comment by czpython** [comment]
Ok to merge

**Pull request merged**